javascript/javascript1/week4/voice.js

```diff
@@ -0,0 +1,54 @@
+const user = " Michael";
+const todos = [];
+
+function getReply(command) {
+  if (command === "Hello my name is" + user) {
+    return "nice to meet you" + user;
+  }
+  if (command === "What is my name?") {
+    return "your name is" + user;
+  }
+
+  if (command === "Add fishing to my todo") {
+    todos.push("fishing");
+    return "fishing is added to your todo list";
+  }
+  if (command === "Add singing in the shower to my todo") {
+    todos.push("singing in the shower");
+    return "added singing in the shower to a list of todos";
+  }
+  if (command === "Remove fishing from my todo") {
+    return "Removed fishing from your todo";
+  }
+  if (command === "What is on my todo?") {
+    return todos;
+  }
+  if (command === "what day is it today?") {
+    return new Date().toLocaleDateString("en-US", {
+      //is not displaying the date after using this method
+      day: "numeric",
+      year: "numeric",
+      month: "long",
+    });
+  }
```

Comment:
Its better to use include( ) for strings instead of ===  operator. 
use different string functions like startsWith( ), endsWith( ) and includes( ) etc  whenever required.